sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -610,9 +610,9 @@ Manage the models stored in your account.
 var client = new DocumentModelAdministrationClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
 // Check number of custom models in the FormRecognizer account, and the maximum number of models that can be stored.
-AccountProperties accountProperties = await client.GetAccountPropertiesAsync();
-Console.WriteLine($"Account has {accountProperties.DocumentModelCount} models.");
-Console.WriteLine($"It can have at most {accountProperties.DocumentModelLimit} models.");
+ResourceDetails resourceDetails = await client.GetResourceDetailsAsync();
```

Comment:
Just double checking, the conclusion was to rename everything that ends with the `Info` suffix to `Details`?

Reply:
I don't remember for sure if we discussed `ResourceInfo` in particular during archboard but Java and .NET got the feedback to rename `Info` => `Details` across the board because we're avoiding the `Info` suffix in general, since it does not convey much meaning.

Reply:
Got it, thanks for the info